### DISCUSSION TITLE
Disable/Hide WalletConnect

### DISCUSSION THREAD

**Discussion by ignatov-dev** [reown-com/appkit#3228] [Q&A]
Hi everyone.

Is it possible to disable WalletConnect?
Setting **enableWalletConnect: false** removes WalletConnect from available connectors but leads to exception when i try to connect other wallets and QR code ui appears.
Seams like it can't work without WalletConnect. So I'm wondering if anybody was able to run it without WalletConnect or without qr code on other wallets

![image](https://github.com/user-attachments/assets/a9af5469-d36c-4c47-9e10-37486fe5ce02)

**Comment by rtomas** [comment]
Hi ! 
I cannot reproduce it ... can you share with me a video or step by step to reproduce it ?

**Comment by ignatov-dev** [comment]
@rtomas Sure. 

On the first step i'm calling createAppkit with this options (enableWalletConnect: false)

```tsx
export const appkit = createAppKit({
  adapters: [wagmiAdapter],
  networks,
  defaultNetwork: bsc,
  projectId,
  metadata,
  enableEIP6963: true,
  enableInjected: true,
  debug: true,
  themeMode: 'light',
  allowUnsupportedChain: false,
  enableWalletConnect: false,
  allWallets: 'HIDE',
  featuredWalletIds: [
    'c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96', // metamask
    '4622a2b2d6af1c9844944291e5e7351a6aa24cd7b23099efac1b2fd875da31a0', // trastwallet
  ],
});
```

Then when I try to connect to MetaMask or Trust Wallet using qr code, I have an error and qr code is not loaded. Providing video of user flow...

https://github.com/user-attachments/assets/7dc65aa5-1a15-44ba-82e6-cb586eef2b80
